### PR TITLE
Fix wrong log

### PR DESCRIPTION
**Pull request by smellthemoon**
Signed-off-by: lixinguo [xinguo.li@zilliz.com](mailto:xinguo.li@zilliz.com)
/kind improvement
/kind bug

### CODE REVIEW

File: internal/querynodev2/handlers.go [hunk 1]
```diff
@@ -326,8 +326,8 @@ func (node *QueryNode) searchChannel(ctx context.Context, req *querypb.SearchReq
 	// get delegator
 	sd, ok := node.delegators.Get(channel)
 	if !ok {
-		err := merr.WrapErrServiceUnavailable("failed to get query shard delegator")
-		log.Warn("Query failed, failed to get query shard delegator", zap.Error(err))
+		err := merr.WrapErrServiceUnavailable("failed to get search shard delegator")
```
Comment (congqixia) [inline internal/querynodev2/handlers.go:329]:
```suggestion
		err := merr.WrapErrServiceUnavailable("failed to get shard delegator for search")
```

**Comment by smellthemoon** [comment]
rerun ut

**Pull request merged**